Tools/Scripts/webkitdirs.pm

```diff
@@ -2632,7 +2632,7 @@ sub shouldUseFlatpak()
         return 0;
     }
 
-    if ((defined $ENV{'WEBKIT_JHBUILD'} && $ENV{'WEBKIT_JHBUILD'}) or defined $ENV{'WEBKIT_BUILD_USE_SYSTEM_LIBRARIES'}) {
+    if ((defined $ENV{'WEBKIT_JHBUILD'} && $ENV{'WEBKIT_JHBUILD'}) or (defined $ENV{'WEBKIT_BUILD_USE_SYSTEM_LIBRARIES'} && $ENV{'WEBKIT_BUILD_USE_SYSTEM_LIBRARIES'})) {
```

Comment:
This is probably an error.

Reply:
The additional expression makes the expression `true` only if WEBKIT_BUILD_USE_SYSTEM_LIBRARIES is defined and its value is true. In other words, when `WEBKIT_BUILD_USE_SYSTEM_LIBRARIES=1`.

Right now, the expression skips flatpak if `WEBKIT_BUILD_USE_SYSTEM_LIBRARIES=0`. In this case, Flatpak SDK should be used (unless WEBKIT_JHBUILD=1).